qgisfeedproject/static/style/bulma.scss

```diff
@@ -0,0 +1,167 @@
+@charset "utf-8";
```

Comment:
@Xpirix, is this a customized Bulma style? Perhaps we should rename it to something different.

Reply:
I mean the file name.

Reply:
@dimasciput , Yes, I've renamed it to custom.bulma.scss